sdk/identity/azure-identity/azure/identity/_credentials/client_secret.py

```diff
@@ -24,10 +24,22 @@ class ClientSecretCredential(ClientCredentialBase):
     :keyword cache_persistence_options: configuration for persistent token caching. If unspecified, the credential
         will cache tokens in memory.
     :paramtype cache_persistence_options: ~azure.identity.TokenCachePersistenceOptions
+    :keyword bool allow_broker:  
```

Comment:
Same comments about the line break formatting and explicit passing of `allow_broker` to the parent class